x-pack/plugins/siem/public/components/flyout/header/index.tsx

```diff
@@ -32,9 +35,8 @@ interface OwnProps {
 interface StateReduxProps {
   description?: string;
   getNotesByIds: (noteIds: string[]) => Note[];
-  history?: History[];
   isFavorite?: boolean;
-  isLive?: boolean;
+  isDatepickerLocked?: boolean;
```

Comment:
This says optional with a `?` but below it's just disregarded

```ts
isDatepickerLocked={isDatepickerLocked!}
```

Removing the `?` and removing the bang seems to be what TypeScript is recommending. I would remove all the optionals and bangs if possible, but up to you.